src/util/webauthn.js

```diff
@@ -63,6 +63,12 @@ function (Okta, Q) {
     getAssertion: getAssertion,
     isAvailable: function () {
       return window.hasOwnProperty('msCredentials');
+    },
+    isNewApiAvailable: function () {
+      return navigator && navigator.credentials && navigator.credentials.create;
+    },
+    isWebauthnOrU2fAvailable: function () {
+      return this.isNewApiAvailable() || window.hasOwnProperty('u2f');
```

Comment:
can we have a function to check `window.hasOwnProperty('u2f')`? It seems that it is used in a few places in the U2FControllers (enroll and verify)... maybe it goes to FidoUtil? Asking so we have only in one place the logic to know if `u2f` is available or not